### DISCUSSION TITLE
Unable to update HomeAssistant addon/update not shown

### DISCUSSION THREAD

**Discussion by m-rossi** [OpenCCU/OpenCCU#2700] [🇺🇸 Q&A] [answered]
### Describe the issue you are experiencing

I just wanted to update my RaspberryMatic HomeAssistant addon from 3.73.9.20240130 to 3.75.6.20240316 but the update is not offered to me.

The supervisor logs shows
```
24-03-26 19:51:33 WARNING (SyncWorker_3) [supervisor.store.data] No repository information exists at /data/addons/git/de838cd8
24-03-26 19:51:34 WARNING (SyncWorker_7) [supervisor.store.data] No repository information exists at /data/addons/git/de838cd8
```

Any advice?

### Describe the behavior you expected

See available update in WebUI/Install the update via WebUI

### Steps to reproduce the issue

1. Go to HA settings
2. Go to Add-Ons
3. Check for updates

### What is the version this bug report is based on?

3.73.9.20240130

### Which base platform are you running?

ha-addon (HomeAssistant Add-on)

### Which HomeMatic/homematicIP radio module are you using?

HM-MOD-RPI-PCB

### Anything in the logs that might be useful for us?

```txt
No
```

### Additional information

The repository is shown as _unknown_:

![Add-on Repositorys verwalten](https://github.com/jens-maus/RaspberryMatic/assets/4509134/92f2ae5b-8f6e-47fd-9e59-31c1dd1ad6fc)

HomeAssistant versions:
- Core 2024.3.3
- Supervisor 2024.03.0
- Operating System 12.1
- Frontend 20240307.0

**Comment by jens-maus** [comment, marked as answer]
Sorry, but this can't be a general issue in RaspberryMatic or its infrastructure. In fact, I tested it myself and I could perfectly add the Add-on repository URL and can also update the Add-on correctly. Thus, I guess this must be a local installation issue only on your side. So please take a sbk-file backup of your Addon, then remove it, remove the add-on store, re-add it and try again.

**Reply by m-rossi** [reply]
It's weird. My HA setup including the HM addon worked perfectly for about a year (thanks for that BTW!). It must be some recent update, may of HA itself, which broke the update mechanism.

**Reply by jens-maus** [reply]
Could be. As said. Simply download a fresh sbk backup, delete the addon, remove the add-on store and re-add it all back...

**Reply by m-rossi** [reply]
I just wanted to report back: Removing and re-adding the addon and repository did it. After restoring the backup everything works like before.

Just to document it somewhere: I did some additional research yesterday and found https://github.com/home-assistant/supervisor/issues/4820, which is similar to the situation I had, especially the output of `ha resolution info`. The creator describes a rather complex solution, which was not necessary in the case of RaspberryMatic.
